### PR TITLE
Added a new connector for Joox, Tencent's music site

### PR DESCRIPTION
**Pull request by mmhan**
Joox can now be scrobbled without any dependency on other third party libraries

### CODE REVIEW

**Comment by mmhan** [comment]
Not quite sure how I'm failing the lint here.

```
Running "unused_files" task
Fatal error: Found 1 unused files: connectors/joox.js
error Command failed with exit code 1.
```

**Comment by alexesprit** [comment]
The `js` property should have `string` type, not `array`.

Also, connector tests are not used anymore.

**Comment by mmhan** [comment]
They have just changed their interface again. Will rework on it.

**Comment by mmhan** [comment]
All changes done. Please check @alexesprit

File: src/connectors/joox.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const playerSelector = '.kIpMXu';
```
Comment (alexesprit) [inline src/connectors/joox.js]: Could you also use strings directly? For example:
```js
Connector.trackSelector = '.jlfeQT';
```

Comment (mmhan) [reply src/connectors/joox.js]: I'd prefer to keep it this way because it's easier to change along with their client-side code.

Comment (alexesprit) [reply src/connectors/joox.js]: I don't know why this way is easier, but the whole PR looks acceptable for me, so I resolve this conversation.

**Pull request merged**